src/profile/v100/entity/persona/persona_data/persona_data.rs

```diff
@@ -147,7 +147,7 @@ impl HasSampleValues for PersonaData {
         Self::new(
             PersonaDataIdentifiedName::sample_other(),
             CollectionOfPhoneNumbers::sample_other(),
-            CollectionOfEmailAddresses::sample(),
+            CollectionOfEmailAddresses::sample_other(),
```

Comment:
Intentional?

Reply:
It looked like an error to me. `sample_other` to use `sample_other`s.